.github/workflows/pull-request-label.yml

```diff
@@ -0,0 +1,166 @@
+name: Pull Request Maintenance
+on:
+  pull_request_target:
+    # make sure that when the PR changes, we also update
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+permissions:
+  pull-requests: write
+  issues: write
+
+jobs:
+  conventional-commit-labeler:
+    name: Label PR based on Conventional Commit Specification
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7
+        env:
+          TYPE_TO_LABEL: |
+            {
+              "feat":"kind/feature",
+              "fix":"kind/bug",
+              "chore":"kind/chore",
+              "ref":"kind/refactor",
+              "clean":"kind/cleanup",
+              "design":"kind/design",
+              "docs":"kind/documentation",
+              "test":"area/testing",
+              "perf":"area/performance"
+            }
+          SCOPE_TO_LABEL: |
+            {
+              "charts":"area/charts",
+              "release":"area/release",
+              "testing":"area/testing",
+              "security":"area/security",
+              "templating":"area/templating",
```

Comment:
I think handling scopes more flexibly might be better. Ideally we want to scope out specific provider features, or core controller features, no?

Reply:
somethign in the lines of  `scopeLabel=area/$SCOPE_TO_LABEL`

Reply:
That would mean that if the label doesn't exist the job fails. Also, it would allow for random scopes where people aren't sure what to use. You know AWS vs aws vs amazon.... :D 

But we can do that if you like.

Reply:
I added dynamic scope handling.